controller-server/src/main/java/com/yahoo/vespa/hosted/controller/application/ApplicationList.java

```diff
@@ -122,8 +122,8 @@ public ApplicationList hasDeployment() {
     }
 
     /** Returns the subset of applications which started failing after the given instant */
-    public ApplicationList startedFailingOnVersionAfter(Version version, Instant instant) {
-        return listOf(list.stream().filter(application -> JobList.from(application).firstFailing().on(version).firstFailing().after(instant).anyMatch()));
+    public ApplicationList startedFailingOnVersionAfter(Version version) {
+        return listOf(list.stream().filter(application -> JobList.from(application).firstFailing().on(version).anyMatch()));
```

Comment:
Impossible to start failing on a version before the version is available. 

I want to remove it because *released at* is a bit imprecise, being the time of the last git commit in the compiled source code, rather than the time of release, as one might be led to believe.